### PR TITLE
added mission 6c nod

### PR DESCRIPTION
**Pull request by 42foobar42**
So here is the last of the level 6 mission.

It is similar to the other 2 mission.

I added a scondary objectiv. But this is very difficult, don't know if it is okay...

### CODE REVIEW

File: mods/cnc/maps/nod06c/map.yaml [hunk 1]
```diff
@@ -0,0 +1,634 @@
...
+	UnitsRallyMiddle: waypoint
+		Location: 55,50
+		Owner: Neutral
+	UnitsEntryRight: waypoint
+		Location: 58,57
+		Owner: Neutral
+	UnitsRallyRight: waypoint
+		Location: 57,51
+		Owner: Neutral
+	ChnEnrty: waypoint
```
Comment (obrakmann) [inline mods/cnc/maps/nod06c/map.yaml]: ChnEntry. Funnily enough, you made the same typo in the lua script as well, so it all works :)

**Comment by obrakmann** [comment]
It's possible to destroy the bridge in this mission, which will prevent you from winning it if you don't have any units on the other side yet. You can fix that by adding this piece of yaml to the rules:
```yaml
^Bridge:
        Invulnerable:
```

The difficulty sure is up a notch, but I think it's fine. Haven't tried to win the secondary objective yet, but I think it should be possible if you use your units wisely.

**Comment by 42foobar42** [comment]
Same typo was in mission 6a, so I fixed it too.

File: mods/cnc/maps/nod06c/map.yaml [hunk 2]
```diff
@@ -0,0 +1,636 @@
...
+	^Ship:
+		Tooltip:
+			GenericVisibility: Enemy
+			ShowOwnerRow: false
+	^Building:
+		Tooltip:
+			GenericVisibility: Enemy
+			ShowOwnerRow: false
+	^Bridge:
+        Invulnerable:
```
Comment (obrakmann) [inline mods/cnc/maps/nod06c/map.yaml]: Indentation is wrong.

**Comment by 42foobar42** [comment]
fixed it

**Comment by Mailaender** [comment]
Tested the winning scenario. :white_check_mark: Well done again.

**Comment by abcdefg30** [comment]
Looks good to me. @obrakmann wanted to take another look and will merge this hopefully then. :)

**Comment by obrakmann** [comment]
:+1: from me as well. And just to confirm that the secondary objective *is* possible:
screenshot from 2015-05-28 00 27 03

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/5949fbfedb366453ec98fa6feccd1ee1e7e76869%5E...5949fbfedb366453ec98fa6feccd1ee1e7e76869)

**Pull request merged**